scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -392,9 +392,26 @@ private void fillVarsPart(String jobDescriptor) {
     }
 
     private Widget prepareWorlflowInformationWidget() {
-        return new HTML("<center> <img src=" + icon + " height=40px width=40px> <br> <b>Documentation :</b> <a href=" +
-                        documentation + " target=_blank>" + documentation + " </a><br> <b> Bucket name :</b> " +
-                        bucketNameGI + " </center>");
+
+        String widgetIcon = "Please add an icon to the workflow <br>";
+        String widgetDocumentation = "Please add a documentation link to the workflow <br>";
+        String widgetBucketName = "Please add a bucket name to the workflow";
```

Comment:
Same comment, is it possible to add bucket name from this interface?

Reply:
I put "No bucketName"